src/main/java/com/marginallyclever/makelangelo/Makelangelo.java

```diff
@@ -180,6 +175,11 @@ private static void setSystemLookAndFeel() {
 	        	UIManager.setLookAndFeel(UIManager.getSystemLookAndFeelClassName());
 	        } catch (Exception e) {}
 		}
+		String os = System.getProperty("os.name").toLowerCase(Locale.ENGLISH);
```

Comment:
this seems pretty self-contained and could probably be in a separate method.

Reply:
ok, done